cfme/utils/smem_memory_monitor.py

```diff
@@ -142,14 +141,15 @@
 
 
 class SmemMemoryMonitor(Thread):
-    def __init__(self, ssh_client, scenario_data):
+    def __init__(self, ssh_client, scenario_data, appliance):
```

Comment:
**REQUIRED:** As we are refactoring this, let's remove the ssh_client argument entirely and just pass the appliance, then we can locally obtain ```ssh_client``` inside this method.